src/main/java/edu/uci/ics/crawler4j/url/WebURL.java

```diff
@@ -67,11 +68,27 @@ public String getURL() {
     }
 
     public void setURL(String url) {
+        boolean validUrl = url.startsWith("//") ||
```

Comment:
I wanted to add, I am not sure if this is appropriate but I think it might be given the parsing logic below.